boards/arm/particle_argon/Kconfig.defconfig

```diff
@@ -0,0 +1,50 @@
+# Kconfig - Particle Argon board configuration
+#
+# Copyright (c) 2018 Matthias Boesl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_PARTICLE_ARGON
+
+config BOARD
+	default "particle_argon"
+
+if ADC
+
+config ADC_0
+	default y
+
+endif # ADC
+
+if I2C
+
+config I2C_0
+	default y
+
+endif # I2C
+
+if SPI
+
+config SPI_2
+	default y
```

Comment:
It would be better to do it as `CONFIG_SPI_2=y` in _particle_argon_defconfig_ since SPI is enabled there. And this would be consistent with Xenon and Boron boards.